fastserde/avro-fastserde/src/main/java/com/linkedin/avro/fastserde/FastGenericDatumReader.java

```diff
@@ -13,12 +14,13 @@
 /**
  * Generic {@link DatumReader} backed by generated deserialization code.
  */
-public class FastGenericDatumReader<T> implements DatumReader<T> {
+public class FastGenericDatumReader<T, U extends GenericData> implements DatumReader<T> {
```

Comment:
We agreed that this breaks the public interface, so the type parameter must be removed.

Reply:
Correct, it's restored in commit
```
[review-fixes] Part 3: Getting rid of additional type-param
in Fast(Reader|Writer) classes: <U extends GenericData>
so that it adheres to standard avro reader/writer classes.
```